### PR TITLE
change dockerfile for ci

### PR DESCRIPTION
**Pull request by gshennvm**
# What does this PR do ?

logic of dockerfile does not change but need to change the ALIGNER_COMMIT arg so it's easier to run on the nemo-ci

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -68,9 +69,13 @@ RUN pip uninstall -y megatron-core && \
     fi && \
     pip install -e .
 
-WORKDIR /opt
-
-# install the latest NeMo-Aligner
-RUN pip install --no-deps git+https://github.com/NVIDIA/NeMo-Aligner.git@main
+RUN git clone https://github.com/NVIDIA/NeMo-Aligner.git && \
```
Comment (trias702) [inline Dockerfile:72]: I would add a comment above `# install the latest NeMo-Aligner`

**Review by trias702 — COMMENTED** [review]
LGTM, but you deleted the comment for the Aligner section and I would recommend adding it back

File: Dockerfile (hunk 1 above)
Comment (gshennvm) [reply Dockerfile:73]: ah right, how's this? 4bac22fdfc57082f0f8bb83c6a80cf8f50f78b2c

**Review by trias702 — APPROVED** [review]
LGTM

**Pull request merged**